dart/lib/src/hub.dart

```diff
@@ -19,7 +18,7 @@ class Hub {
   final ListQueue<_StackItem> _stack = ListQueue();
 
   /// if stack is empty, it throws IterableElementError.noElement()
-  _StackItem _peek() => _stack.isNotEmpty ? _stack.first : null;
+  _StackItem? _peek() => _stack.isNotEmpty ? _stack.first : null;
```

Comment:
ideally this would be non null, we'd need to adjust the code similar to https://github.com/getsentry/sentry-java/blob/main/sentry/src/main/java/io/sentry/Stack.java#L69-L74